x-pack/plugins/security_solution/public/cases/components/connectors/case/alert_fields.tsx

```diff
@@ -27,9 +27,7 @@ const Container = styled.div`
 
 const defaultAlertComment = {
   type: CommentType.generatedAlert,
-  alerts: '{{context.alerts}}',
-  index: '{{context.rule.output_index}}',
-  ruleId: '{{context.rule.id}}',
+  alerts: `[{{#context.alerts}}{"_id": "{{_id}}", "_index": "{{_index}}"}__SEPARATOR__{{/context.alerts}}]`,
```

Comment:
If we add a `,` instead of a `__SEPARATOR__ ` will that work? This will remove the need for us to replace it on the backend.